### DISCUSSION TITLE
Instructions to build from sources

### DISCUSSION THREAD

**Discussion by tiix95** [FlareSolverr/FlareSolverr#1284] [Ideas]
Hi,

First, thanks a lot for the work.

Could you please consider adding build instructions inside readme ?
I use pre-compiled binaries on my server to avoid installing all python dependencies, but due to issue #1253, it is not working any more. I'd like to be able to compile by myself on development VM after applying proposed patch waiting for a new release.

Thank you very much !

**Comment by ilike2burnthing** [comment]
They're there, always have been.
